### DISCUSSION TITLE
Slickgrid dataview column id

### DISCUSSION THREAD

**Discussion by senaerg** [ghiscoding/slickgrid-react#58] [Q&A]
When I try to use webpack slickgrid lib with a table that doesn't have id as a column but some other column name as an  identifier, I get error that id doesn't  exist. I don't explicitly call dataview in the code. How can I resolve it?

**Comment by ghiscoding** [comment]
Every data array must include unique `id` property, if you want to use a different property name you can use `datasetIdPropertyName` grid option to change `id` to something else. Basically SlickGrid requires unique ID in your array, it will not work without this condition met, this also means that you cannot use an array of just strings or numbers, it only works with an array of objects.
